src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key in actions) {
```

Comment:
you don't have to iterate over `actions` with for-in loop, use `forEach`

Reply:
Hi! I have corrected the task js_stateful-objects according to your suggestions, but I won't use "forEach" as you suggested. I tried but I cannot. That is in a future module (Array Iteration Method), but I am still at the "Object Advanced" module. I hope the "for in" will work for now. Thank you for your understanding.